app/views/RoomView/index.tsx

```diff
@@ -1361,8 +1361,8 @@ class RoomView extends React.Component<IRoomViewProps, IRoomViewState> {
 		if (showUnreadSeparator || dateSeparator) {
 			return (
 				<>
-					<Separator ts={dateSeparator} unread={showUnreadSeparator} />
 					{content}
+					<Separator ts={dateSeparator} unread={showUnreadSeparator} />
```

Comment:
This is needed in order to show the separator on the right place.
Since we're not using the same inverting list technique anymore, we need to reposition.